cdec/installation-manager-core/src/test/java/com/codenvy/im/managers/TestConfigManager.java

```diff
@@ -146,7 +149,8 @@ public void testLoadDefaultMultiServerCdecConfig() throws Exception {
     @Test(expectedExceptions = IOException.class,
             expectedExceptionsMessageRegExp = "Can't download installation properties. error")
     public void testLoadDefaultCdecConfigTransportError() throws Exception {
-        doThrow(new IOException("error")).when(transport).download(endsWith("codenvy-multi-server-properties/3.1.0"), any(Path.class));
+        doThrow(new IOException("error")).when(transport).download(endsWith("codenvy-multi-server-properties/3.1.0"), any(Path.class),
+                                                                   MediaType.APPLICATION_OCTET_STREAM);
```

Comment:
Leads to _Mockito_ error `Unfinished stubbing detected`.
Fixup: `eq(MediaType.APPLICATION_OCTET_STREAM)`